examples/isostasy_calculation.py

```diff
@@ -0,0 +1,54 @@
+"""
+Calculation of
+"""
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import harmonica as hm
+
+# Load the elevation model
+data = hm.datasets.fetch_topography_earth()
+print(data)
+
+# Define the density values
+density_upper_crust = 2670
+density_lower_crust = 2800
+density_mantle = 3300
+density_oceanic_crust = 2900
+density_water = 1000
+
+# Root calculation considering the ocean
+root = hm.isostasy_airy(data.topography.values,
+                        density_upper_crust,
+                        density_lower_crust,
+                        density_mantle,
+                        density_oceanic_crust=density_oceanic_crust,
+                        density_water=density_water)
```

Comment:
This looks really good! You could probably set the densities directly in the function like:

```
root = hm.isostasy_airy(data.topography.values,
                        density_upper_crust=2670,
                        density_lower_crust=2800,
                        density_mantle=3300,
                        density_oceanic_crust=2900,
                        density_water=1000)
```